lambdas/file_access_patterns/lambda_for_fileDelete/index.js

```diff
@@ -16,31 +16,34 @@ function response(statusCode, message) {
 }
 
 exports.deleteFile = async (event) => {
-  const reqBody = event.body;
+  const reqBody = JSON.parse(event.body);
```

Comment:
don't need to parse..the api gateway is configured to parse and send the event.

Reply:
Okay